setup.py

```diff
@@ -24,24 +24,27 @@ def exec_file(names):
 
 
 setup(
-    name="matrix_client",
-    version=exec_file(("matrix_client", "__init__.py",))["__version__"],
-    description="Client-Server SDK for Matrix",
-    long_description=read_file(("README.rst",)),
-    author="matrix.org",
-    author_email="team@matrix.org",
-    url="https://github.com/matrix-org/matrix-python-sdk",
+    name='matrix_client',
+    version=exec_file(('matrix_client', '__init__.py',))['__version__'],
+    description='Client-Server SDK for Matrix',
+    long_description=read_file(('README.rst')),
+    author='The Matrix Developers (matrix.org)',
```

Comment:
I see, I wrote "The Matrix Developers" as it seems to be a common author name for projects with many contributors (see Rust for example). I'll go ahead and change it though if you'd like.